httpclient5/src/main/java/org/apache/hc/client5/http/async/methods/BasicHttpRequests.java

```diff
@@ -40,6 +42,38 @@
  */
 public final class BasicHttpRequests {
 
+    // TODO Next version of HttpCore:
+    // Method.normalizedValueOf(method)
+    private static Method normalizedValueOf(final String method) {
+        return Method.valueOf(Objects.requireNonNull(method, "method").toUpperCase(Locale.ROOT));
```

Comment:
@garydgregory Please use `Args.notNull` for consistency with the rest of the code base.

Reply:
Done.